SimG4Core/PhysicsLists/plugins/QGSPCMS_FTFP_BERT.cc

```diff
@@ -18,9 +19,18 @@ QGSPCMS_FTFP_BERT::QGSPCMS_FTFP_BERT(const edm::ParameterSet& p) : PhysicsList(p
   bool hadPhys = p.getUntrackedParameter<bool>("HadPhysics", true);
   bool tracking = p.getParameter<bool>("TrackingCut");
   double timeLimit = p.getParameter<double>("MaxTrackTime") * CLHEP::ns;
+  double minFTFP = p.getParameter<double>("EminFTFP") * CLHEP::GeV;
+  double maxBERT = p.getParameter<double>("EmaxBERT") * CLHEP::GeV;
+  double minQGSP = p.getParameter<double>("EminQGSP") * CLHEP::GeV;
```

Comment:
@bsunanda , eminQGSP is fixed in any way to 12 GeV, addition of it differently inside CMSSW is possible but only open extra freedom.